totalRP3/modules/register/main/register_exchange.lua

```diff
@@ -49,13 +49,14 @@ local displayMessage = TRP3_API.utils.message.displayMessage;
 -- WoW imports
 local UnitName, UnitIsPlayer, UnitFactionGroup, CheckInteractDistance, UnitFullName = UnitName, UnitIsPlayer, UnitFactionGroup, CheckInteractDistance, UnitFullName;
 local tinsert, time, type, pairs, tonumber = tinsert, GetTime, type, pairs, tonumber;
-local after = C_Timer.After;
+local after = C_Timer.NewTimer;

```

Comment:
It would be better to have our local reference be newTimer so that it is clear in that we are not actually using after here. We may need to use after in this file at some point.